### DISCUSSION TITLE
How to display Formula correctly in ui.markdown

### DISCUSSION THREAD

**Discussion by xiaozhiagi** [zauberzeug/nicegui#1311] [Q&A]
### Question

In ui.markdown, Formula such as $5 text {mm} $cannot be rendered and displayed normally. How can I solve this problem? Thank you

**Comment by falkoschindler** [comment]
I don't know. Since `ui.markdown` is based on https://pypi.org/project/markdown2/, it won't support LaTeX unless markdown2 does.

**Comment by rodja** [comment]
There is feature request #696. I'll close this discussion. Please place any further comments on the feature request.
